cli/connhelper/commandconn/commandconn.go

```diff
@@ -0,0 +1,270 @@
+// Package commandconn provides command-based net.Conn.
+// Package commandconn does not depend on docker CLI.
```

Comment:
It would be nice if this explained how to use this package, what the protocol is, and why one might do it.